config/config.go

```diff
@@ -4,6 +4,7 @@ import (
 	apiv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"time"
```

Comment:
you need to run `goimports` on this file to correct the formatting

Reply:
ran goimports and it reformatted the file